README.md

```diff
@@ -70,36 +72,30 @@ Currently, testing with external projects is somewhat inelegant, but this repo i
   }
 }
 ```
-- Gatsby and React both rely on some globally-persisted values which can cause errors if you have multiple copies of these dependencies installed. When testing a Gatsby site, many issues can be worked around by temporarily deleting the `demo-gatsby` package and bootstrapping again.
 
+- Gatsby and React both rely on some globally-persisted values which can cause errors if you have multiple copies of these dependencies installed. When testing a Gatsby site, many issues can be worked around by temporarily deleting the `demo-gatsby` package and bootstrapping again.
 
 ## Release Process
 
-Tina has three main branches:
+Tina has two main branches:
 
 - **master:** The bleeding edge of tinacms
-- **next:** A preview of the next release
 - **latest:** The current stable release
 
 The flow of changes therefore looks like:
 
-> `fix-some-bug` => `master` => `next` => `latest`
+> `fix-some-bug` => `master` => `latest`
 
-The process happens over a week:
+This is a weekly process:
 
-- On Monday
-  1. `next` is merged into `latest`; then `latest` is published to npm
-  2. `master` is merged into `next`; then `next` is published to npm
-- Any hot fixes for bugs will be cherry picked into `next` and `latest`
-  and the published accordingly.
-- Every pull request merged to `master` automatically triggers a
-  `canary` release.
+- On Monday `master` is merged into `latest` which is then published to npm.
+- Hot fixes are cherry picked onto `latest` and the published.
+- Prereleases are created off of `master` whenever they're needed.
 
 With this process:
 
-- all accepted changes are available as `canary` releases for early testing
 - critical fixes are published as soon as possible
-- new features and minor fixes take ~1.5 weeks to be published
+- new features and minor fixes take ~0.5 week to be published
```

Comment:
```suggestion
- new features and minor fixes take ~4 days to be published
```

using fractions-of-a-week as a time interval feels a little funky. how do you feel about this?